supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -358,7 +408,7 @@ Use all lower case, unless used in a title or at the beginning of a sentence.
 // Ceph: General; kept as is
 [[shard-n]]
 ==== image:images/yes.png[yes] shard (noun)
-*Description*: A database _shard_ is a horizontal partition of data in a database or search engine. Each individual partition is referred to as a shard or database shard. Each shard is held on a separate database server instance, to spread load.
+*Description*: 1) A database _shard_ is a horizontal partition of data in a database or search engine. Each individual partition is referred to as a _shard_ or _database shard_. To spread load, each shard is held in a separate database server instance. 2) In storage, a _shard_ is a small part of a larger container. Shards replicate independently in parallel with other shards. Operations on shards take less time than operations on the whole container, which makes replication and maintenance more reliable.
```

Comment:
```suggestion
*Description*: A database _shard_ is a horizontal partition of data in a database or search engine. To spread load, each shard is held in a separate database server instance. In storage, a _shard_ is a small part of a larger container. Shards replicate independently in parallel with other shards. Operations on shards take less time than operations on the whole container, which makes replication and maintenance more reliable.
```

Reply:
Updated the numbering; kept that sentence